### ISSUE TITLE
Enable accessibility by default

### ISSUE THREAD

**Issue by ethindp** [brave/brave-browser#7650]
## Description 

Currently, the Chromium browser only enables accessibility on Windows and Mac OS, and only if a screen reader is determined to be running at the time the chromium engine performs the check. The reasoning, at least according to the Electron documentation, is that enabling accessibility by default causes a performance hit. I have found no evidence to support a claim. I do not believe accessibility effects the browser experience for people who are not using a screen reader. Please enable accessibility by default, whether a screen reader is running or not, so that people with disabilities on platforms such as Linux can use it.

## Steps to Reproduce

   1. On Linux, install Brave and the Orca screen reader.
   2. Launch Orca and Brave.
   3. Try to navigate the brave window.

## Actual result:

Orca is unable to interact with the Brave window whatsoever (other than reading the title bar). The Chromium browser accepts the option "--force-renderer-accessibility", however Brave seems to ignore this flag and accessibility is still not enabled.

## Expected result:
Orca should be able to interact fully with the Brave window (including add-on windows that use the Brave HTML renderer).

## Reproduces how often: 

This problem reproduces itself every time Orca is running and Brave is launched.

## Brave version (brave://version info)

Brave Browser 79.1.2.41

## Version/Channel Information:

Stable channel
- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the dev channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? N/A
- Does the issue resolve itself when disabling Brave Rewards? N/A
- Is the issue reproducible on the latest version of Chrome? Yes, except that --force-renderer-accessibility does enable accessibility (though there are still accessibility problems).

**Comment by westes** [comment]
I have downloaded brave 87.1.8. I am running Ubuntu 18.04 under gnome/orca. The behavior is that orca reacts to and reads controls as I tab between them but I do not get any feedback when using arrow keys to read text on a web page or in the address bar.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
